browser/net/brave_ad_block_csp_network_delegate_helper.h

```diff
@@ -0,0 +1,27 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_NET_BRAVE_AD_BLOCK_CSP_NETWORK_DELEGATE_HELPER_H_
+#define BRAVE_BROWSER_NET_BRAVE_AD_BLOCK_CSP_NETWORK_DELEGATE_HELPER_H_
+
+#include <memory>
+
+#include "brave/browser/net/url_context.h"
+#include "net/base/completion_once_callback.h"
+#include "net/http/http_response_headers.h"
+#include "url/gurl.h"
```

Comment:
it seems to me most of the parameter types can be forward declarations, so most of includes can go to .cc